test/Microsoft.ML.CodeAnalyzer.Tests/Helpers/DiagnosticVerifier.cs

```diff
@@ -267,7 +268,7 @@ private static string FormatDiagnostics(DiagnosticAnalyzer analyzer, params Diag
 
         private static readonly MetadataReference MSDataDataViewReference = RefFromType<IDataView>();
         private static readonly MetadataReference MLNetCoreReference = RefFromType<IHostEnvironment>();
-        private static readonly MetadataReference MLNetDataReference = RefFromType<Model.ModelLoadContext>();
+        private static readonly MetadataReference MLNetDataReference = RefFromType<KeyToValueMappingTransformer>();
```

Comment:
>KeyToValueMappingTransformer [](start = 83, length = 28)

maybe MlContext?

Reply:
Done!

---
In reply to: [255255322](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255255322) [](ancestors = 255255322)